canteens/bsr.py

```diff
@@ -0,0 +1,108 @@
+import datetime
+import re
+import urllib.request
+
+
+from bs4 import BeautifulSoup
+from celery.utils.log import get_task_logger
+
+from backend.backend import app, cache, cache_date_format, cache_ttl
+from canteens.canteen import VEGGIE, MEAT, get_next_week, get_current_week
+
+import emoji
+WELLBALANCEDMEAL = emoji.emojize(':green_apple:')
+
+logger = get_task_logger(__name__)
+import logging as logger
+
+URL = 'https://www.bsr.de/bsr/speiseplan/internet_speiseplan.html'
+
+
+def get_website():
+    return urllib.request.urlopen(URL).read()
+
+
+def main(date):
+    dishes = []
+
+    menus = BeautifulSoup(get_website(), 'html5lib').find_all('table', class_='Liste') #Note: Only html5lib can fix this broken html code.
+    days  = BeautifulSoup(get_website(), 'html5lib').find_all('div', class_='KopfLeiste_o')
```

Comment:
Please do not download the same website twice. The bot already sends about 1000 HTTP requests during an update run. There is no need to call get_website() twice.